### DISCUSSION TITLE
API roadblock

### DISCUSSION THREAD

**Discussion by Sternbach-Software** [Sternbach-Software/GeniForAndroid#17] [General]
@whittede any update on the geni API?

**Comment by whittede** [comment]
Hey @Sternbach-Software ! Thanks for reaching out!

I think currently I am still waiting for [this pull request](https://github.com/Sternbach-Software/GeniForAndroid/pull/16) to be merged. (Which, based on your last comment, I believe you said looks alright?)

Regarding getting the API to return specific values as we were discussing in [this issue](https://github.com/Sternbach-Software/GeniForAndroid/issues/9), I have a reminder for myself to continue working on it that is set for November 28th. (Coincidentally, tomorrow!) I'll be flying home to Oregon tomorrow, so likely it will be in the next few weeks that I'll have some update to talk about. I'll let you know how it goes soon!
